vtex-apps/admin-pages
# Allow editing URL

#### What is the purpose of this pull request?

Adds the ability to navigate to a specific URL by editing the storefront URL in the UI

#### What problem is this solving?

Allow users to access a specific page easily

#### How should this be manually tested?
1- Go to /admin/cms/storefront
2- Enter an entire URL of the store, or just the pathname, into the URL
input
3- Verify that you are redirected to the right page

#### Screenshots or example usage
![url navigation](https://user-images.githubusercontent.com/1354492/59643671-f459be00-913f-11e9-85ac-7761a84dfaf0.gif)


#### Types of changes
